util/execdetails/execdetails.go

```diff
@@ -189,17 +187,23 @@ type CopDetails struct {
 	RocksdbBlockReadCount uint64
 	// RocksdbBlockReadByte is the total number of bytes from block reads.
 	RocksdbBlockReadByte uint64
+	// WaitWallTimeMs is the off-cpu wall time which is elapsed in TiKV side. Usually this includes queue waiting time and
+	// other kind of waitings in series.
+	ProcessTime time.Duration
+	WaitTime    time.Duration
```

Comment:
How about add `RocksdbDetail` struct and add `String` method for it, Since rockdb information will display in `CopRuntimeStats` and `SnapshotRuntimeStats`, just use a common method to make samw.

```go
type ScanDetail struct {
	// TotalKeys is the approximate number of MVCC keys meet during scanning. It includes
	// deleted versions, but does not include RocksDB tombstone keys.
	TotalKeys int64
	// ProcessedKeys is the number of user keys scanned from the storage.
	// It does not include deleted version or RocksDB tombstone keys.
	// For Coprocessor requests, it includes keys that has been filtered out by Selection.
	ProcessedKeys int64

	RocksdbDetail
}

type RocksdbDetail struct {
	// RocksdbDeleteSkippedCount is the total number of deletes and single deletes skipped over during
	// iteration, i.e. how many RocksDB tombstones are skipped.
	RocksdbDeleteSkippedCount uint64
	// RocksdbKeySkippedCount it the total number of internal keys skipped over during iteration.
	RocksdbKeySkippedCount uint64
	// RocksdbBlockCacheHitCount is the total number of RocksDB block cache hits.
	RocksdbBlockCacheHitCount uint64
	// RocksdbBlockReadCount is the total number of block reads (with IO).
	RocksdbBlockReadCount uint64
	// RocksdbBlockReadByte is the total number of bytes from block reads.
	RocksdbBlockReadByte uint64
}
```

Reply:
How about just use ScanDetailV2 structure?

Reply:
Do you mean adding a `String` method for `ScanDetailV2` ? @breeswish

Reply:
ScanDetailV2 is implemented by Protobuf. I suggest to add a function accepting it as a parameter.

Reply:
I am confused the meaning of `add a function accepting it as a parameter`? Which function do you mean?

Reply:
Do you think we can replace `ScanDetail` with `ScanDetailV2` generated by pb in `util/execdetails/execdetails.go` directly?

Reply:
Should extract `TimeDetail` and `ScanDetailV2` in `ExecDetail`, just like the pb?